src/FixtureBuilder/ExpressionLanguage/Parser/TokenParser/Chainable/SimpleReferenceTokenParser.php

```diff
@@ -41,7 +41,9 @@ public function parse(Token $token): FixtureReferenceValue
         $value = $token->getValue();
 
         try {
-            return new FixtureReferenceValue(substr($value, 1));
+            $elements = empty($value) ? false : substr($value, 1);
```

Comment:
Is this necessary? IIRC this parser can only parse `SIMPLE_REFERENCE_TYPE` tokens which have the form `@.+`, i.e. `$value` is a non-empty string here and starts with `@`. Anything else IMO should go with the exception.

Looking at it again though it might be clearer to throw the exception if `$value` is not a string and `$elements` an empty string rather than relying on an exception to be thrown and catch it:

```php

$value = $token->getValue();

if (!is_string($value) || '' === $value) {
    throw ExpressionLanguageExceptionFactory::createForUnparsableToken($token);
}

return new FixtureReferenceValue(substr($value, 1));
```